crates/turborepo-filewatch/src/lib.rs

```diff
@@ -86,10 +84,30 @@ pub struct FileSystemWatcher {
     // dropping the other sender for the broadcast channel, causing all receivers
     // to be notified of a close.
     _exit_ch: tokio::sync::oneshot::Sender<()>,
+    pub cookie_dir: AbsoluteSystemPathBuf,
 }
 
 impl FileSystemWatcher {
-    pub async fn new(root: &AbsoluteSystemPath) -> Result<Self, WatchError> {
+    pub async fn new_with_default_cookie_dir(
+        root: &AbsoluteSystemPath,
+    ) -> Result<Self, WatchError> {
+        // We already store logs in .turbo and recommend it be gitignore'd.
+        // Watchman uses .git, but we can't guarantee that git is present _or_
+        // that the turbo root is the same as the git root.
+        Self::new(root, &root.join_components(&[".turbo", "cookies"])).await
+    }
```

Comment:
why do we need a new method instead of just inlining into `new()`?

Reply:
The filewatcher is somewhat generic so I didn't want to fully hard-code where the cookie files should go.